AssetsPickerViewController/Classes/Photo/Controller/AssetsPhotoViewController.swift

```diff
@@ -161,6 +161,8 @@ open class AssetsPhotoViewController: UIViewController {
                 self.delegate?.assetsPickerCannotAccessPhotoLibrary?(controller: self.picker)
             }
         }
+      
+        cameraPicker.setAllowsEditing(shouldAllow: pickerConfig.allowsEditing)
```

Comment:
is it safe to assume a picker config exists are this point? seems other uses of it optionally bind

Reply:
@alanipce I think so yes. There is only one spot the AssetsPhotoViewController gets initialized and immediately after a non-optional AssetsPickerConfig is passed to the assetphoto vc's picker config property. So I think it is safe to assume that a picker config exists inside the viewDidLoad. 

Additionally, looking at the extension file with +Setup, those functions (such as setupBarButtonItems) are also directly calling on pickerConfig w/out unwrapping so I think we should be good.